_data/publications.yml

```diff
@@ -1,3 +1,9 @@
+ - title: "Volumetric Data Fusion of External Depth and Onboard Proximity Data For Occluded Space Reduction"
+   authors: Matthew Strong*, Caleb Escobedo*, Alessandro Roncone
```

Comment:
What's the asterisk about ? Equal contribution? Just leave it without asterisk.

Reply:
Fixed